doc/introduction/inspecting_circuits.rst

```diff
@@ -153,6 +163,9 @@ results.
 2: array([0.707+0.j, 0.+0.j, 0.+0.j, 0.707+0.j]),
 'execution_results': 0.0}
 
+All snapshots are numbered with consecutive integers, and if no tag was provided,
+the number of a snapshot is used as key in the output instead.
```

Comment:
I thought this might be a helpful comment here.

Reply:
```suggestion
All snapshots are numbered with consecutive integers, and if no tag was provided,
the number of a snapshot is used as a key in the output dictionary instead.
```